modules/nodes-pods-autoscaling-more-about.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-about.adoc
+
+:_content-type: CONCEPT
+[id="nodes-pods-autoscaling-more-about_{context}"]
+= More about horizontal pod autoscalers
+
+The horizontal pod autoscaler (HPA) extends the concept of pod auto-scaling. The HPA lets you create and manage a group of load-balanced nodes. The HPA automatically increases or decreases the number of pods when a given CPU or memory threshold is crossed.
+
+== How does the HPA work?
```

Comment:
Each section heading should have a unique ID/anchor added. Here and throughout for the other sections.

Reply:
Yeah I missed that , good catch!